RELEASE.md

```diff
@@ -1,11 +1,10 @@
-
-To create a release, run the following:
+To create a release, update the version number in `nbclassic/__version__.py`, then run the following:
 
 ```
-git clean -dfx
+git clean -dfxx
```

Comment:
Is one `x` superfluous?

Reply:
No, it's extra assurance that *everything* is removed.

Reply:
Wait, a while ago I looked up the options and that extra x was necessary. I can't find it in the manual now.

Reply:
Ah, it's `-dffx`, not `-dfxx`.